playbooks/cloud-provider-openstack-acceptance-test-manila-provisioner/run.yaml

```diff
@@ -0,0 +1,93 @@
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-k8s
+    - clone-devstack-gate-to-workspace
+    - role: install-devstack
+      environment:
+        OVERRIDE_ENABLED_SERVICES: 'key,c-sch,c-api,c-vol,rabbit,mysql'
```

Comment:
Whether we need to enable manila and ceph in devstack when we run manila-provisioner? if yes, you can use role `create-devstack-local-conf`, like this:

```yaml
  roles:
    - config-golang
    - install-k8s
    - clone-devstack-gate-to-workspace
    - create-devstack-local-conf
      enable_services:
        - 'manila'
        - 'ceph'
    - role: install-devstack
```

Reply:
Thanks a lot!